src/formatDate.js

```diff
@@ -8,7 +8,54 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const [oldData, idFromYear, oldYear, fromSumbol] = getFormat(fromFormat);
```

Comment:
The variable `fromSumbol` is misspelled. It should be `fromSymbol` to correctly represent the delimiter symbol in the date format.